core/db/migrate/20210815004823_add_unique_index_to_option_values_variants.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddUniqueIndexToOptionValuesVariants < ActiveRecord::Migration[5.2]
+  def up
+    remove_index :spree_option_values_variants, [:variant_id, :option_value_id]
+    add_index :spree_option_values_variants, [:variant_id, :option_value_id],
+              name: "index_option_values_variants_on_variant_id_and_option_value_id",
```

Comment:
out of curiosity, why the explicit index name?

Reply:
Rails will generate an index name that exceeds the 64 character limit. The name I've used just matches the existing index's name, so presumably whoever created the original index also ran into this.